content/software-security/compliance/cmmc-2/cmmc-practices.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Overview of CMMC 2.0 Practices/Control Groups "
+description: "Overview of CMMC 2.0 Practices/Control Groups "
+lead: "Learn about the 14 differenct domains of practices required for CMMC 2.0."
+type: "article"
+date: 2024-08-09T19:10:09+00:00
+lastmod: 2024-08-09T19:10:09+00:00
+contributors: ["Erin Glass"]
+draft: false
+tags: ["compliance", "CMMC 2.0", "standards"]
+images: []
+menu:
+  docs:
+    parent: "cmmc-2"
+weight: 003
+toc: true
+---
+
+In total, Cybersecurity Maturity Model Certification (CMMC) 2.0 has 130 practices, though only Level 3 needs to meet them all. These practices are divided into 14 domains, each of which covers a different aspect of cybersecurity.
```

Comment:
This sounds like Level 3's requirements are known whereas previous pages sound like they are still being determined.

Reply:
Good catch!